src/js/forms.js

```diff
@@ -99,7 +97,7 @@ class Forms {
 		this.validateInput(input);
 	}
 
-	handleClickEvent(event) {
+	static handleClickEvent(event) {
```

Comment:
>Expected 'this' to be used by class method 'handleClickEvent'.

@JakeChampion I followed your lead on making these static methods to get around the linting issue. It seems sensible but as it effects the api of an oForm instance it is technically a breaking change. I'm not worried in this case but thought if you hadn't realised it was worth mentioning incase it is an issue.

Reply:
Very good point, we should instead disable the eslint error for these methods and create an issue to move these to static in the next major